src/libs/actions/Report.ts

```diff
@@ -1174,6 +1174,10 @@ function saveReportDraftComment(reportID: string, comment: string | null) {
     Onyx.merge(`${ONYXKEYS.COLLECTION.REPORT_DRAFT_COMMENT}${reportID}`, prepareDraftComment(comment));
 }
 
+function saveReportDraftCommentWithCallback(reportID: string, comment: string | null, callback: () => void) {
+    Onyx.merge(`${ONYXKEYS.COLLECTION.REPORT_DRAFT_COMMENT}${reportID}`, prepareDraftComment(comment)).then(callback);
+}
+
```

Comment:
Do we need a separate method for callback alone? 

Can we add the callback to existing method and by default callback param empty if not passed?

Reply:
@Santhosh-Sellavel I updated.